test/clojure-mode-sexp-test.el

```diff
@@ -117,6 +117,16 @@ and point left there."
     (clojure-backward-logical-sexp 1)
     (should (looking-at-p "\\^String biverse"))))
 
+(ert-deftest test-sexp-with-namespaced-map ()
+  (with-temp-buffer
+    (insert "first #:name/space{:k v}")
+    (clojure-mode)
+    (clojure-backward-logical-sexp 1)
+    (should (looking-at-p "#:name/space{:k v}"))
+    (insert " #::ns {:k v}")
```

Comment:
Why is there are space here between the namespace and the map?

Reply:
Both `#:ns {:k v}` and `#:ns{:k v}` are valid syntax and has the same meaning. Also, I have not seen any guideline that prefers one style over the other.

Reference:
https://dev.clojure.org/jira/browse/CLJ-1910?focusedCommentId=42834&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-42834